src/kohlrahbi/ahb/__init__.py

```diff
@@ -3,6 +3,7 @@
 """
 
 import fnmatch
+import gc
```

Comment:
is this a core module?

Reply:
Yes. But calling it manually is kind of a code smell